### DISCUSSION TITLE
allow for policy OCI bundle download url environment variable

### DISCUSSION THREAD

**Discussion by mightyBroccoli** [aquasecurity/trivy#4807] [Ideas]
### Description

Many corporate networks do not allow tools to download freely from the internet.
Would it be possible to allow the policy bundle to also be cacheable with a customizable Repository URL ?

Something like TRIVY_POLICY_BUNDLE_URL 

We are currently caching the trivy_db and the trivy_java_db with a Habor pull-through proxy for our internal CI/CD system.

### Target

None

### Scanner

Misconfiguration

**Comment by simar7** [comment]
Already being tracked here https://github.com/aquasecurity/trivy/issues/4672
